proj/hps_accel/Makefile

```diff
@@ -74,7 +74,7 @@ EXTRA_LITEX_ARGS += --cpu-variant slim+cfu
 
 # gen2 needs special build args and only works on hps
 ifeq '$(GATEWARE_GEN)' '2'
-EXTRA_LITEX_ARGS += --separate-arena --cfu-mport --cpu-variant=slimopt+cfu
+EXTRA_LITEX_ARGS += --separate-arena --cfu-mport
```

Comment:
This looks unrelated to this commit, and maybe you didn't intend to include it?

Reply:
Thanks. Removed.